### ISSUE TITLE
Can not send mail with attachments

### ISSUE THREAD

**Issue by bolasblack** [jorgemanrubia/mailgun_rails#29]
Hey jorgemanrubia, , so glad to find mailgun_rails support attachments now.

Today I tried to send a mail with attachment, but I failed……

Then I found the code https://github.com/jorgemanrubia/mailgun_rails/blob/master/lib/mailgun/attachment.rb#L13 will raise a `TypeError`, message is `no implicit conversion of Hash into String`

I'm not sure what's wrong……

My ruby version is 2.2.0, and my code is here:

```ruby
  def destroyed user
    @user = user
    attachments['exported_data.json'] = {
      mime_type: 'application/json',
      content: user.export_data.to_json
    }
    mail from: '...', to: '...'
  end
```

**Comment by onemanstartup** [comment]
I found a bug, I'm sorry, when I was testing, I was testing only inline attachments and they worked.
@bolasblack Please could you use this https://github.com/onemanstartup/mailgun_rails/tree/attachments_fix branch and see if it was fixed. Thank you.

**Comment by bolasblack** [comment]
@onemanstartup I will have a try right now, thank you

**Comment by bolasblack** [comment]
@onemanstartup Awesome! The bug has fixed! Thanks!

**Comment by jorgemanrubia** [comment]
Released 0.6.5 containing the patch. Thanks for reporting the bug and thank you so much @onemanstartup for fixing it.

**Comment by bolasblack** [comment]
It's so great to see this PR has been merged, thanks for your work @jorgemanrubia @onemanstartup !
